catalogue/webapp/pages/work.tsx

```diff
@@ -1,40 +1,29 @@
 import { GetServerSideProps, NextPage } from 'next';
-import {
-  Work as WorkType,
-  CatalogueApiError,
-} from '@weco/common/model/catalogue';
-import ErrorPage from '@weco/common/views/components/ErrorPage/ErrorPage';
+import { Work as WorkType } from '@weco/common/model/catalogue';
 import Work from '../components/Work/Work';
 import { getWork } from '../services/catalogue/works';
 import {
   getGlobalContextData,
   WithGlobalContextData,
 } from '@weco/common/views/components/GlobalContextProvider/GlobalContextProvider';
 import { removeUndefinedProps } from '@weco/common/utils/json';
-import { appError, AppErrorProps } from '@weco/common/views/pages/_app';
+import {
+  appError,
+  AppErrorProps,
+  WithPageview,
+} from '@weco/common/views/pages/_app';
 
 type Props = {
-  workResponse: WorkType | CatalogueApiError;
-} & WithGlobalContextData;
+  workResponse: WorkType;
+} & WithGlobalContextData &
+  WithPageview;
 
 export const WorkPage: NextPage<Props> = ({
   workResponse,
   globalContextData,
 }: Props) => {
-  return (
-    <>
-      {workResponse.type === 'Error' && (
-        <ErrorPage
-          statusCode={workResponse.httpStatus}
-          title={workResponse.description}
-          globalContextData={globalContextData}
-        />
-      )}
-      {workResponse.type !== 'Error' && (
-        <Work work={workResponse} globalContextData={globalContextData} />
-      )}
-    </>
-  );
+  // TODO: move the <Work> ocmponent content back into here
```

Comment:
can get rid of this comment

Reply:
The `TODO` was added as I didn't want to clutter this PR any more. It wasn't very clear though, is [this](https://github.com/wellcomecollection/wellcomecollection.org/pull/5819/commits/ff4586a1d9addfed46fce17f25de8c407bf24c92) better?

Reply:
yep, misunderstood that